added Pursuance FAQ section (Issue #13)

howmanywhat opened the pull request:
Please review style/format, if everything looks okay, there are still a couple questions currently having answers "TODO" which need authoritative answers (re: blockchain and infiltration) before merging.

Also note that the answer for "Why should someone use Pursuance?" came from the gist /pursuance-status.md -- not currently set for public viewing(?), so this can be changed/removed or, if it's suitable for the public, a "For more details see [link-to-gist]" can be added.

howmanywhat committed:
suscribe button changes, deleted changed lines

howmanywhat committed:
added Pursuance FAQ section, text mod, a'ed q's, accordion collapse, colorsplash gif

elimisteve commented on index.html:
```diff
@@ -151,8 +285,8 @@ <h4>Artists and creative types</h4>
                   <h4>Join our Mailing List!</h4>
-                  <p>Our monthly mails include updates on the platform's development and ways to get involved. <a class="text-primary" href="#mc_embed_signup" data-toggle="modal">Sign up here!</a></p>
-                  <button class="subscribe-bottom"><a href="#mc_embed_signup" data-toggle="modal" class="nav-link">Subscribe to our mailing list here!</a></button>
+                  <p>Our monthly mails include updates on the platform's development and ways to get involved.</p>
```
Let's not say monthly, as it could be more or less frequent than that.

howmanywhat replied:
Will change "Our monthly mails include" to "Our mailings include"

Also here's another q/a to be added to the faq if it's okay --

**Q**: When will the Pursuance Project be ready for the general public to use?
**A**: An ambitious list of features and components integration is planned for the foreseeable future starting with:
- Closed Alpha testing (through June/July 2018)
- Closed Beta testing (August through October 2018)
- **Public v1.0 launch** (late October / early November 2018)
- Continued integration and adoption of latest technologies, as they become available, for Pursuance v2.0 (mid 2019)
- Security audit of the web app and native apps, complete secure integration across all social media platforms, continued iteration over all aspects of the UX (2019 - 2021)

Note - that answer's paraphrased from the gist /pursuance_feature_roadmap.md (which has a github tag "secret") so is it okay to phrase it that way and/or include a "For feature/component details see [link-to-gist]" line after the bullet points?

elimisteve reviewed (changes requested):
@geistling What else should go into the FAQ?

elimisteve commented:
@aburkha4 Want to add to the FAQ?

howmanywhat commented:
added another commit for review -- changed "monthly mails" to "mailings", wrote answers for the 2 todo questions, added q/a for the roadmap, added js to collapse current accordion panel when another one is selected, changed bg-colorsplash to a gif with the two twitter banners crossfading into each other

elimisteve commented:
We should add an answer to https://twitter.com/willbradley/status/1026678950775291904 to the FAQ
